.eslintrc

```diff
@@ -260,6 +260,6 @@
     "import/no-duplicates": 2,
     "import/newline-after-import": 2,
     "import/unambiguous": 2,
-    "import/extensions": [ 1, "always" ]
+    "import/extensions": [ 0, "always" ]
```

Comment:
Yes, but when you use a linter in an editor it uses it's own modules, not the local install ones. You must update your global eslint-plugin-import.

When you run `npm run lint-js` do you get the same error?